### PR TITLE
feat: allow show/hide meters of each category from the category menu.

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#268

### CODE REVIEW

**Comment by ceoaliongroo** [comment]
__Update:__
screen shot 2015-02-08 at 4 22 35 pm

**Comment by ceoaliongroo** [comment]
__Update:__ filter only in the principal state
screen shot 2015-02-08 at 5 03 09 pm
screen shot 2015-02-08 at 5 03 00 pm

File: client/app/views/dashboard/main.categories.html [hunk 1]
```diff
@@ -1,33 +1,38 @@
...
-    </li>
-  </ol>
-</ul>
+<form name="categoryMenuForm">
+  <ul ui-tree data-drag-enabled="false" data-empty-place-holder-enabled="false" class="tree">
+    <ol ui-tree-nodes="">
+      <li ng-repeat="category in categories.tree" ui-tree-node ng-show="hasMeters(category)">
+        <div ui-tree-handle class="tree-level-top">
+          <a ui-sref="dashboard.withAccount.categories({categoryId: category.id})" ng-class="{'active-category':  ('categoryId' | includedByParams:category.id) }">
+            <i class="glyphicon"></i>
```
Comment (amitaibu) [inline client/app/views/dashboard/main.categories.html]: remove

**Comment by ceoaliongroo** [comment]
Merged.

**Pull request closed without merging**